src/connectors/pagination/Paginator.js

```diff
@@ -13,7 +13,9 @@ class Paginator {
     if (total === 0) return [0];
 
     const totalDisplayedPages = this.nbPagesDisplayed(padding, total);
-    if (totalDisplayedPages === total) return range(0, total);
+    if (totalDisplayedPages === total) {
+      return range({ start: 0, end: total });
```

Comment:
should we consistently leave out start = 0 or consistently pass it?

Reply:
Good point – I can remove it here.